CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# Unreleased
+
+New features:
```

Comment:
Since we are fixing an existing feature this should be a fix and patch versioned release.

Reply:
Changed to `Bug fixes:`